cc/mcts_player.cc

```diff
@@ -34,8 +34,7 @@ std::ostream& operator<<(std::ostream& os, const MctsPlayer::Options& options) {
      << " soft_pick:" << options.soft_pick
      << " random_symmetry:" << options.random_symmetry
      << " resign_threshold:" << options.resign_threshold
-     << " batch_size:" << options.batch_size
-     << " komi:" << options.komi
+     << " batch_size:" << options.batch_size << " komi:" << options.komi
```

Comment:
So this is driving me bananas but some part of the fancy google magic vim thing is insisting on autoformatting on save.